client/src/components/admin/AdminRound/AdminRoundContent.jsx

```diff
@@ -86,11 +86,11 @@ function AdminRoundContent({ round, competitionId, officialWorldRecords }) {
     onError: apolloErrorHandler,
   });
 
-  function handleResultAttemptsSubmit(attempts) {
+  function handleResultAttemptsSubmit(attempts, person) {
     if (isBatchMode) {
       setBatchResults([
         ...batchResults.filter((result) => result.id !== editedResult.id),
-        { id: editedResult.id, attempts, enteredAt: nowISOString() },
+        { id: editedResult.id, attempts, person, enteredAt: nowISOString() },
```

Comment:
I don't think we should be storing the whole `person` object in local storage.

What we could do is, right before calling `attemptResultsWarning`, we can build a copy of `results` with changes applied. We can map over `results` and look for a batch result entry with a matching id, if the is one, we take its attempts.

One issue with checking against batch results is that once we show the warning, it may be confusing, because there will be no visible results for the conflicting person (they are in the batch). But I guess it's better than not showing the warning.

Reply:
Sounds good!  I decided to create a new `combinedResults` array in the `AdminRoundContent` component so that it can be recalculated only when it needs to change.  Let me know if that looks good to you; I'm not a React expert by any means!

I also realized that this warning would pop up if you pulled up a result and tried to enter it without changing anything.  I saw a couple people using this functionality to refresh `results` data without needing to refresh the entire page.  In the interest of not breaking current users' workflows, [I opted to filter out the current result from the duplicate check](https://github.com/thewca/wca-live/pull/282/commits/4177f552794aff566770a41432448ba41454ac7f).